ApolloExtra.lua

```diff
@@ -2587,10 +2823,24 @@ OlympusLootData.ApolloUpgrade = {
 				RequiredMaxHealthFraction = 0.20,
 				RequiredMaxLastStands = 1,
 
-				{ Cue = "/VO/Apollo_0078",
+				{ Cue = "/VO/Apollo_0082",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Emote = "PortraitEmoteSurprise",					
+					Text = "{#DialogueItalicFormat}Ohh{#PreviousFormat}, that must hurt. Like really, really hurt. I don't think it think it will make the difference this time, but it's better than nothing." },
```

Comment:
"{#DialogueItalicFormat}Ohh{#PreviousFormat}, that must hurt. Like really, really hurt. I don't think it think it will make a difference this time, but it's better than nothing."